### DISCUSSION TITLE
Please add support for the new Button entity type

### DISCUSSION THREAD

**Discussion by blalor** [zachowj/node-red-contrib-home-assistant-websocket#530] [Feature Request]
Home Assistant added a new "button" entity type. Is it possible to get support for this added for use in node-red alongside the switch, binary_sensor, and sensor entity types?

**Comment by zachowj** [comment]
It's been added to the custom component. Now, a node just needs to be created.

**Reply by zachowj** [reply]
Added in v0.41.0

**Reply by blalor** [reply]
Thank you! I was confused when I couldn't configure ha-entity to be a button but I see that ha-button now exists. Is that just while it's in beta?

**Reply by zachowj** [reply]
No, all new entities will be getting their own node and existing ones will get migrated over at some point.
